applications/data_analysis/spectral_analysis.ipynb

```diff
@@ -0,0 +1,2142 @@
+{
```

Comment:
<p>Small thing, but I think there should be a comma after "currently".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>